lesson14/index.html

```diff
@@ -0,0 +1,31 @@
+<!DOCTYPE html>
+<html lang="ja">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <meta http-equiv="X-UA-Compatible" content="ie=edge" />
+    <link rel="stylesheet" href="./style.css"  />
+    <script src="./main.js" defer></script>
+    <title>TerraceTechフロントエンドエンジニア養成所</title>
+  </head>
+  <body>
+    <div>
```

Comment:
Q. 
Is this empty `<div>` necessary?

Reply:
Thank you for your feedback Haru-san,
the <div> is not necessary, so I removed that.
Please see the commit 
10d6a13925dbfcfc0fd346260585132aa1dd91ed